pontoon/base/static/js/translate.js

```diff
@@ -1566,6 +1566,19 @@ var Pontoon = (function (my) {
 
 
     /*
+     * Prevent too long translation submissions
+     */
+    translationLengthLimitExceeded: function (translation) {
+      if (this.translationLengthLimit && this.stripHTML(translation).length > this.translationLengthLimit) {
+        this.endLoader('Translation too long.', 'error');
```

Comment:
Also, I'd move a side-effect of this function outside and leave a simple:
`return this.translationLengthLimit && this.stripHTML(translation).length > this.translationLengthLimit`